packages/js-plugin-nft-storage/src/utils.ts

```diff
@@ -58,7 +58,7 @@ export async function toEncodedCar(
 ): Promise<EncodedCar> {
   await blockstore.put(block.cid, block.bytes);
   const car = new BlockstoreCarReader([block.cid], blockstore);
-  const cid = block.cid;
+  const {cid} = block;
```

Comment:
Can we enforce spaces within an object?

```suggestion
  const { cid } = block;
```

Reply:
this one fixed with prettier